Python/crossinterp.c

```diff
@@ -0,0 +1,617 @@
+
+/* API for managing interactions between isolated interpreters */
+
+#include "Python.h"
+#include "pycore_ceval.h"         // _Py_simple_func
+#include "pycore_crossinterp.h"   // struct _xid
+#include "pycore_pyerrors.h"      // _PyErr_Clear()
+#include "pycore_pystate.h"       // _PyInterpreterState_GET()
+#include "pycore_weakref.h"       // _PyWeakref_GET_REF()
+
+
+/**************************/
+/* cross-interpreter data */
+/**************************/
+
+/* cross-interpreter data */
```

Comment:
Looks like a duplicate of the above comment

Reply:
Thanks for pointing this out.  It was intentional, but the distinction wasn't particularly clear (obviously).  I've fix up the grouping, with explicit labels.